samples/python/django/sample.py

```diff
@@ -0,0 +1,145 @@
+import os
+import sys
+import datetime
+import django
+import pytz
+from django.db import connection
+from django.db.transaction import atomic
+from django.db import transaction
+x = 0
+
+def get_singer(singer_id):
+  global x
+  x += 1
+  return Singer(id=singer_id, first_name='singer', last_name=str(x), full_name= 'singer'+str(x), active=True, created_at=datetime.datetime.now(pytz.UTC), updated_at=datetime.datetime.now(pytz.UTC))
+
+def get_album(album_id, singer=None):
+  global x
+  x += 1
+  return Album(id=album_id, singer=singer, title='album'+str(x), marketing_budget = 200000, release_date = datetime.date.today(), cover_picture= b'hello world',created_at=datetime.datetime.now(pytz.UTC), updated_at=datetime.datetime.now(pytz.UTC))
+
+def get_track(track_id, track_number, album = None):
+  global x
+  x += 1
+  return Track(track_id=track_id, album=album, track_number=track_number, title='track'+str(x), sample_rate=124.543, created_at=datetime.datetime.now(pytz.UTC), updated_at=datetime.datetime.now(pytz.UTC))
+
+def get_venue(venue_id):
+  global x
+  x += 1
+  return Venue(id=venue_id, name='venue'+str(x), description='description'+str(x), created_at=datetime.datetime.now(pytz.UTC), updated_at=datetime.datetime.now(pytz.UTC))
+
+def get_concert(concert_id, venue = None, singer = None):
+  global x
+  x += 1
+  return Concert(id=concert_id, venue=venue, singer=singer, name='concert'+str(x), start_time=datetime.datetime.now(pytz.UTC), end_time=datetime.datetime.now(pytz.UTC)+datetime.timedelta(hours=1), created_at=datetime.datetime.now(pytz.UTC), updated_at=datetime.datetime.now(pytz.UTC))
+
+
+def create_tables():
+  file = open('create_data_model.sql', 'r')
+  ddl_statements = file.read()
+  with connection.cursor() as cursor:
+    cursor.execute(ddl_statements)
+
+@atomic(savepoint=False)
```

Comment:
Is `savepoint=False` required? I would have thought that it was the default.

Reply:
By default, `savepoint=True` is there but that doesn't matter in this case as it is a simple atomic block. `savepoints` come into picture in the case of nested atomic blocks and for nested atomic blocks, we will have to manually do `savepoints=False`. 

In this particular case, we can remove `savepoint=False` and it will work fine. But I thought of including it as it will remind people that in general, savepoints are not supported by us

Reply:
OK, makes sense.